### PR TITLE
Create lightning nodes world heat map (clearnet)

### PR DESCRIPTION
**Pull request by nymkappa**
This PR is a redo of https://github.com/mempool/mempool/pull/2129 using apache echart.

It adds a new chart component which shows where clearnet lightning nodes are mostly concentrated. The map is interactive as well so you can filter countries using the heat map at the bottom right. Clicking on a country which have at least 1 node will open the country lightning nodes list page.

Screen Shot 2022-07-20 at 11 31 47 AM

### CODE REVIEW

File: frontend/src/app/components/graphs/graphs.component.html [hunk 1]
```diff
@@ -38,6 +38,8 @@
         i18n="lightning.nodes-per-isp">Lightning nodes per ISP</a>
       <a class="dropdown-item" routerLinkActive="active" [routerLink]="['/graphs/lightning/nodes-per-country' | relativeUrl]"
         i18n="lightning.nodes-per-isp">Lightning nodes per country</a>
+      <a class="dropdown-item" routerLinkActive="active" [routerLink]="['/graphs/lightning/nodes-map' | relativeUrl]"
+        i18n="lightning.nodes-per-isp">Lightning nodes world map</a>
```
Comment (softsimon) [inline frontend/src/app/components/graphs/graphs.component.html:42]: Need unique i18n string.

File: frontend/proxy.conf.js [hunk 2]
```diff
@@ -85,7 +85,7 @@ if (configContent && configContent.BASE_MODULE == "liquid") {
     });
 } else {
     PROXY_CONFIG.push({
-        context: ['/resources/pools.json', '/resources/assets.json', '/resources/assets.minimal.json'],
+        context: ['/resources/pools.json', '/resources/assets.json', '/resources/assets.minimal.json', '/ressources/worldmap.json'],
```
Comment (softsimon) [inline frontend/proxy.conf.js:88]: ressources -> resources spelling. But why fetch the worldmap json from mempool.space when using local dev? Shouldn't the worldmap json data just be accessed from the local resources folder.

Comment (wiz) [inline frontend/proxy.conf.js]:
```suggestion
        context: ['/resources/pools.json', '/resources/assets.json', '/resources/assets.minimal.json'],
```

**Review by wiz — APPROVED** [review]
Tested ACK @ v2.5.0-dev [[e0cc58bc](https://github.com/mempool/mempool/commit/e0cc58bc)]

**Pull request merged**